packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1235,6 +1265,7 @@ export class Cluster extends ClusterBase {
    * This is because the values is only be retrieved by the API and not exposed
    * by CloudFormation. If this cluster is not kubectl-enabled (i.e. uses the
    * stock `CfnCluster`), this is `undefined`.
+   * The same as IssuerUrl
```

Comment:
But its not, it differs from it slightly by not starting with `https/http`. Lets drop this.

Reply:
removed this line